.github/workflows/tutorials.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       if: runner.os == 'macOS'
       run: |
         if [ "${{ matrix.compiler }}" = "gcc-9" ]; then
-          brew install gcc
+          brew install gcc@9
```

Comment:
out of curiosity, why the change?

Reply:
brew now installs gcc 10 by default, was easier to fix just this line than the other ones for gcc 10.